### PR TITLE
DE560 Adding JWT signing algorithm configuration

### PR DESCRIPTION
**Pull request by bethrichardson**

### CODE REVIEW

**Comment by bethrichardson** [comment]
@arcivanov Review and merge please

File: hcsos.config [hunk 1]
```diff
@@ -20,7 +20,8 @@ PG_PORT=5432
 BACKEND_HOST="$INTERNAL_HOST"
 BACKEND_PORT=8200
 
-SECURITY_MODEL="pbkdf2:sha256"
+HASH_MODEL="pbkdf2:sha256"
```
Comment (arcivanov) [inline hcsos.config]: This is `user password hash algorithm`

File: hcsos.config [hunk 2]
```diff
@@ -20,7 +20,8 @@ PG_PORT=5432
...
+SIGN_MODEL="HS256"
```
Comment (arcivanov) [inline hcsos.config]: this is `JWT algorithm`

File: hcsos.config [hunk 3]
```diff
@@ -20,7 +20,8 @@ PG_PORT=5432
...
 SECURITY_SALT_LENGTH=8
```
Comment (arcivanov) [inline hcsos.config]: this is `user password hash salt`

File: hcsos.config [hunk 4]
```diff
@@ -20,7 +20,8 @@ PG_PORT=5432
...
 SESSION_EXPIRATION=900
```
Comment (arcivanov) [inline hcsos.config]: This is `JWT token expiration` (no sessions)

File: hcsos.config [hunk 5]
```diff
@@ -20,7 +20,8 @@ PG_PORT=5432
...
 SECRET_KEY="225ed702003ab81aef91b539e3b146e1a0da557ad43fda91ed50f9627e0b351f"
```
Comment (arcivanov) [inline hcsos.config]: This is `JWT HMAC secret key`

**Comment by arcivanov** [comment]
Please group the JWT security parameters and user password hash parameters together into different paragraphs

**Comment by arcivanov** [comment]
This shall be merged!

**Pull request merged**